BacktraceScreen.h

```diff
@@ -0,0 +1,47 @@
+#ifndef HEADER_BacktraceScreen
+#define HEADER_BacktraceScreen
+/*
+htop - BacktraceScreen.h
+(C) 2023 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
```

Comment:
This should be put in the `.c` file. Reasoning see in the previous comment in `Action.h`.